nodejs/index.ts

```diff
@@ -48,25 +48,34 @@ async function retrieveAppInfo(client): Promise<IotCentralClient> {
 }
 
 async function updateApp(client): Promise<IotCentralClient> {
-    const result = await client.apps.update(RESOURCEGROUPNAME, RESOURCENAME, UPDATEAPP);
+    const result = await client.apps.beginUpdateAndWait(RESOURCEGROUPNAME, RESOURCENAME, UPDATEAPP);
     console.log(result);
     return new Promise<IotCentralClient>(resolve => resolve(client));
 }
 
 async function listAllAppsByResourceGroup(client): Promise<IotCentralClient> {
-    const result = await client.apps.listByResourceGroup(RESOURCEGROUPNAME);
+    const result = [];
```

Comment:
does it not already return an array of resource groups?

Reply:
in the new generation JS SDks, we will return a iterable iterator for list operations.